web/src/components/LoginGroupError.js

```diff
@@ -11,7 +11,12 @@ const LoginLocked = () => {
             <span>Job Code Missing</span>
           </h1>
           <p>
-            You don’t have the correct job code to access the eAPD system. The <a href={`${apiUrl}/docs/account-registration`}>account registration guide</a> will help you with instructions on getting the correct job code in your EUA account.
+            You don’t have the correct job code to access the eAPD system. The{' '}
+            <a href={`${apiUrl}/docs/account-registration`}>
+              account registration guide
+            </a>{' '}
```

Comment:
I've been using `&nbsp;` good to know about this: `{' '}`

Reply:
yeah, apparently prettier does that for you